crates/sui-indexer/src/handlers/checkpoint_handler.rs

```diff
@@ -146,6 +172,77 @@ where
         })
     }
 
+    // TODO(gegaowp): temp. solution for slow object commits, will be removed after.
+    async fn start_download_and_index_object(&self) -> Result<(), IndexerError> {
+        info!("Indexer object checkpoint download & index task started...");
```

Comment:
this is not right in case the whole indexer restarts, this will fetch from latest "tx checkpoint" instead of "obj checkpoint", but to unblock this is okay.